File: objc/Project8/Project8/WindowController.m
//
//  WindowController.m
//  Project8
//
//  Created by Jinwoo Kim on 1/3/21.
//

#import "WindowController.h"

@interface WindowController ()

@end

@implementation WindowController

- (void)windowDidLoad {
    [super windowDidLoad];
    [self.window setStyleMask:self.window.styleMask | NSWindowStyleMaskFullSizeContentView];
    self.window.titlebarAppearsTransparent = YES;
    self.window.titleVisibility = NSWindowTitleHidden;
    [self.window setMovableByWindowBackground:YES];
}

@end

File: swift/Project8/Project8/WindowController.swift
//
//  WindowController.swift
//  Project8
//
//  Created by Jinwoo Kim on 1/3/21.
//

import Cocoa

class WindowController: NSWindowController {

    override func windowDidLoad() {
        super.windowDidLoad()
        window?.styleMask = [window!.styleMask, .fullSizeContentView]
        window?.titlebarAppearsTransparent = true
        window?.titleVisibility = .hidden
        window?.isMovableByWindowBackground = true
    }

}
